src/config/AppConstant.js

```diff
@@ -0,0 +1,9 @@
+const API_BASE_URL = "http://127.0.0.1:8000/";
```

Comment:
The `csrftoken` and `sessionid` cookies would not set in my browser unless the `API_BASE_URL` was `"http://localhost:8000/"`